Dockerfile

```diff
@@ -0,0 +1 @@
+FROM mongo:latest
```

Comment:
I'm honestly not convinced we need this seeing as this mongo-latest should be interchangeable with whatever we tag this image as but as promised, a dockerfile (which may be useful in the future, idk)